graphs/kickiip.fbp

```diff
@@ -1,4 +1,5 @@
-# !runtime noflo
+# @runtime noflo-nodejs
+# @name IIPKickTest
```

Comment:
@bergie can you please document these things in CHANGES.md? otherwise how are people to know...